packages/ds-healthcare-gov/src/styles/components/_Footer.scss

```diff
@@ -14,48 +17,41 @@
   }
 }
 
-.hc-c-footer__inline-item {
-  color: var(--color-muted);
-  // We use a right margin and padding to avoid a weird
-  // left indent when an item wraps to the next line
-  margin-right: $spacer-half;
+.hc-c-footer__list {
+  align-items: center;
+  display: flex;
```

Comment:
using flex on the ul/list itself helps with the weird spacing issues encountered when the links wrapped to new lines - removed padding/margin hack for flex.